### DISCUSSION TITLE
Possibility to restrict orders per E-Mail address

### DISCUSSION THREAD

**Discussion by choseh** [pretix/pretix#2488] [Q&A] [answered]
Hi,

is it possible to limit orders to, say, one per e-mail address?

Thanks

**Comment by Ahrdie** [comment, marked as answer]
You can limit the maximum amount a product can be bought in the product's availability tab.
I assume you mean to limit it across multiple orders?
May I ask why this should be a viable way to hinder someone from just creating another mail address and place another order?

**Reply by raphaelm** [reply]
We agree that this is a requirement that does not really make sense because it is too easy to create new email addresses.

That said, due to customer pressure, it is possible to do this as a plugin and we offer such a plugin to pretix Enterprise customers: https://marketplace.pretix.eu/products/emailrestrictions/
